services/src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -34,8 +34,8 @@
 public class Utils {
   private static final Logger LOG = LoggerFactory.getLogger(Utils.class);
 
-  private static final int ZK_CONNECTION_TIMEOUT_MS = 30_000;
-  private static final int ZK_SESSION_TIMEOUT_MS = 30_000;
+  public static final int ZK_CONNECTION_TIMEOUT_MS = 30_000;
+  public static final int ZK_SESSION_TIMEOUT_MS = 30_000;
```

Comment:
Can we also simplify the code in method `createMonitoringTopicIfNotExists()` by removing check for method `partitionFactor <= 0`? We can validate the config in XXXConfig class. And `AdminUtils.createTopic(...)` will throw exception if partition number <= 0.